src/test/java/com/ccsu/cs/tutoring/BinaryTreeTest.java

```diff
@@ -0,0 +1,48 @@
+package com.ccsu.cs.tutoring;
+
+import com.ccsu.cs.tutoring.structures.BinaryTree;
+import org.junit.jupiter.api.Test; // for Test annotations
+import org.junit.jupiter.api.Assertions; // for tests themselves
+
+public class BinaryTreeTest {
```

Comment:
Add the following tests to make sure the structure of the tree is created properly:

- deleting leaf nodes reduces size appropriately
- deleting branch nodes replaces them with the proper descendent node
- inserting nodes changes the tree's height appropriately
- traversing in different ways returns proper strings